src/app/_components/DistrictSearch/DistrictList.js

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import { Tick } from '@icons/index';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import 'swiper/css';
+import { PillButton } from '@/app/_components';
+
+export function DistrictList({ list, className }) {
+  const [selected, setSelected] = useState(0);
+  const handleClick = index => {
+    setSelected(index);
+  };
+
+  return (
+    <Swiper slidesPerView="auto" spaceBetween={15} wrapperClass={className}>
+      {list.map(({ id, name }, index) => {
+        const isSelected = index === selected;
+        const hasIcon = isSelected ? <Tick /> : null;
+
+        return (
+          <SwiperSlide key={id} onClick={() => handleClick(index)} className="fit-co">
```

Comment:
fit-co -> fit-content?

Reply:
it shouldn't be there at all, removed it